conneg-by-ap/index.html

```diff
@@ -1027,8 +1033,10 @@ <h4>HTTP Alternate Representations Data Model Implementation</h4>
           </ul>
           <p>
             Using these <code>Link</code> HTTP header parts, this HTTP Functional Profile is able to communicate
-            alternate profiles for a resource as per the <a href="#altr">Alternate Profiles Data Model</a> with the
-            mapping given in <a href="#fig-altr-link-header-mapping"></a> below.
+            alternate profiles for a resource as per the <a href="#altr">Alternate Profiles Data Model</a> and
+            <a href="#fig-altr-link-header-mapping"></a> below maps the header parts to the
+            <a href="#altr-owl">OWL ontology</a> representation of the Alternate Profiles Data Model which is a concrete
+            and formal realization of it. All Functional Profiles should do as per <a href="#abstractmodel"></a>.
```

Comment:
"All functional Profiles should do as per abstract model". When I read this, I don't understand what the functional profiles must do. Do they need to conform to the abstract model?

Reply:
@larsgsvensson: I obliquely referred to the directive in the Abstract Model that:

> implementers SHOULD demonstrate conformance of their Alternate Representations Data Model implementations by providing a published mapping between their environment-specific realization of the model and a concrete realization of the model given in this specification, such as its OWL ontology representation

I've now made that explicit by quoting the directive.